### DISCUSSION TITLE
Can you add a fully liberated or under enemy control symbol or color?

### DISCUSSION THREAD

**Discussion by HezBen** [vonbeardy/hd2galaxy-issues#52] [Ideas]
I am being a huge nerd about planning my attacks, and where our lines are the weakest, and it would be really useful if I could tell at a glance whether or not a planet is under super earth control or not without clicking on it. Maybe just color automaton planets red, bug planets yellow and super earth planets green? And planets that are being contested and fought over can stay gray? 
 
I’d love to see this, but I don’t know how hard it would be or if anyone else would find it useful. Ty for all your work!

**Comment by vonbeardy** [comment]
I've had the same thought about this but was torn on the implementation. If/when I do add this I'll probably make it a setting to enable/disable so it doesn't clutter the screen any more.
